square/gradle-dependencies-sorter
# Add `--version` flag for version verification

### Summary

This pull request adds a `--version` flag to the CLI, allowing users to retrieve the version of the local artifact. This is a follow-up to [#92](https://github.com/square/gradle-dependencies-sorter/pull/92).

### Motivation

Our project's pre-commit hooks use this CLI to sort dependencies in changed Gradle modules. To ensure consistency, part of our hook logic checks that the plugin version matches the version installed on the local machine. If the version specified in the `.toml` file is newer than the local version, it will trigger a download of the updated version.

To support this functionality, the CLI needs a way to report the version of the local artifact, which is now made possible with the `--version` flag.

### Testing

1. Publish the artifact to the local Maven repository:
   ```
   $ ./gradlew publishToMavenLocal
   ```
2. Navigate to the local Maven directory:
   ```
   $ cd ~/.m2/repository/com/squareup/sort-gradle-dependencies-app/0.11-SNAPSHOT
   ```
3. Run the following command to verify the `--version` flag:
   ```
   $ java -jar sort-gradle-dependencies-app-0.11-SNAPSHOT-all.jar --version
   ```
